components/CheckDocumentation/vacuum/TxidWraparound.tsx

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+
+import {
+  CheckDocs,
+  CheckGuidanceProps,
+  CheckTriggerProps,
+} from "../../../util/checks";
+import { useSmartAnchor } from "../../SmartAnchor";
+
+const TxidWraparoundTrigger: React.FunctionComponent<CheckTriggerProps> = ({
+  config,
+}) => {
+  const warning = config.settings["warning_threshold_pct"] as number;
+  const critical = config.settings["critical_threshold_pct"] as number;
+  return (
+    <p>
+      Detects when the transaction ID utilization on the server is more than{" "}
+      <code>{warning}%</code> and creates an issue with severity "warning".
+      Escalates to "critical" once the utilization exceeds{" "}
+      <code>{critical}%</code>. Resolves once the utilization falls below the
+      threshold.
+    </p>
+  );
+};
+
+const TxidWraparoundGuidance: React.FunctionComponent<CheckGuidanceProps> = ({
+  urls: { backendsUrl },
+}) => {
+  const Link = useSmartAnchor();
+  return (
+    <div>
+      <h4>Impact</h4>
```

Comment:
Stepping back from commenting on specific copy, I think we want to discuss a few things here:
 - the TXID/MXID counter is approaching wraparound
 - vacuum normally addresses this through freezing but may not be keeping up
   - may want to mention anti-wraparound vacuums
 - if this is not addressed, the database will eventually shut down to prevent corruption and require manual intervention
 - Postgres 14+ gives you failsafe autovacuum, which is an additional safeguard against this

So I think the docs below are roughly the right structure, but:
 - "transaction ID usage" is sort of ambiguous: users could interpret that to mean allocation
   - we could use "transaction ID counter is approaching wraparound" as in my MXID suggestion above (this still uses the term "wraparound" which I had said I wanted to avoid in the description, but I think it's okay as long as we're not implying that wraparound will happen)
   - we could talk about the "transaction ID space utilization" -- this may be a little too abstract, but the Postgres docs do mention "xid space" and it's sometimes used to discuss these concepts
 - we should avoid implying that anti-wraparound vacuums are the only thing that help reclaim TXIDs/MXIDs